react-app/src/view/Question/ListOfQuestions.jsx

```diff
@@ -27,28 +56,61 @@ export default function ListOfQuestions() {
     getQuestions();
   }, []);
 
+  const theme = createTheme({
+    components: {
+      MuiButton: {
+        styleOverrides: {
+          root: {
+            borderRadius: 8,
+            "&:hover": {
+              backgroundColor: " #f2f2f2",
+              color: "#333333",
+            },
+            backgroundColor: " #100F0F",
+            float: "right",
+            color: "#00",
+          },
+        },
+      },
+    },
+  });
```

Comment:
You can put this in the global theme that you have. No need to put it here and wrap another Theme provider around the button if you want all the buttons to look like that automatically